.github/workflows/ci.yml

```diff
@@ -39,6 +39,7 @@ jobs:
 
       - name: Test
         run: pytest --doctest-modules -v --ignore=scrap $PROJECT_NAME
+        
   publish:
     name: Publish
     if: "!contains(github.event.head_commit.message, '[skip ci]') && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main')"
```

Comment:
What's this do?

Reply:
only run the job if on master (or main) and if `[skip ci]` is not part of the commit message